backends/utils.py

```diff
@@ -2,12 +2,18 @@
 Utility functions for the backends
 """
 from datetime import datetime, timedelta
+import logging
 import pytz
 
+from django.core.exceptions import ObjectDoesNotExist
```

Comment:
maybe it's because I could not figure out a specific one there. feel free to leave as is